build.xml

```diff
@@ -18,22 +21,73 @@
     <mkdir dir="${dist.lib.ext.dir}"/>
   </target>
 
+  <!-- builds the server extension tar for use with extension loader -->
+  <target name="tar" depends="oauth-social-jar,oauth-social-common-jar" description="Creates the tar file">
+    <mkdir dir="${build.dir}/zm-oauth-social"/>
+    <mkdir dir="${build.dir}/zm-oauth-social/ext"/>
+    <mkdir dir="${build.dir}/zm-oauth-social/ext-common"/>
+    <copy file="${build.dir}/${oauth-social.jar.file}" tofile="${build.dir}/zm-oauth-social/ext/zmoauthsocial.jar"/>
+    <copy file="${build.dir}/${oauth-social-common.jar.file}" todir="${build.dir}/zm-oauth-social/ext-common"/>
+    <tar destfile="${build.dir}/zm-oauth-social.tar">
+      <tarfileset dir="${build.dir}/zm-oauth-social" prefix="/zm-oauth-social" includes="**"/>
+    </tar>
+  </target>
+
   <!-- builds the server extension jar -->
   <target name="jar" depends="compile,zimbra-jar" description="Creates the jar file" />
 
+  <target name="oauth-social-jar" depends="compile"
+       description="Creates the zm-oauth-social.jar file which is used by for oauth functionality">
+     <antcall target="zimbra-jar">
+       <param name="implementation.title" value="Zimbra Oauth Social Client Jar"/>
+       <param name="jar.file" value="${oauth-social.jar.file}"/>
+       <param name="includes"
+         value="com/zimbra/oauth/handlers/impl/**,
+         com/zimbra/oauth/resources/**,
+         com/zimbra/oauth/schedulers/impl/**,
```

Comment:
These weren't brought over (for obvious compatibility reasons), I'm assuming the task ignores them.

Reply:
No sure, will ping you later

Reply:
Talked with  Stephan, this port is good for calssic